apis/betterself/v1/correlations/views.py

```diff
@@ -89,6 +96,13 @@ def get(self, request):
         if aggregate_dataframe.empty:
             return NO_DATA_RESPONSE
 
+        if cumulative_lookback > 1:
```

Comment:
Eventually bring this over to the other views and then one that will just output to xlsx!